README.md

```diff
@@ -86,10 +85,20 @@ npm install
 npm run start
 ```
 
-----
+## Loki
 
-Brought to you by [Prezly][prezly].
+We use [Loki](https://loki.js.org/) visual testing library for storybook.
+There are several commands that you can use:
+
+1. `loki` - checks that your current stories are matched what you already have (Storybook server should run)
+1. `loki:update` - Regenerate all snapshots despite what you already have (Storybook server should run)
+1. `loki:static-build` - Performs checks like `loki` does, but over builded sources from storybook, no need to run Storybook server
 
+If Loki fails on CI you can download generated images like [here](https://github.com/actions/upload-artifact#where-does-the-upload-go)
```

Comment:
Indeed, where does it go? 😄

Reply:
It creates zip file which you can download on actions page like in the example here https://github.com/actions/upload-artifact#where-does-the-upload-go